<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nodeAdmin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="users">
         <attribute name="title">
          <string>Users</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTreeWidget" name="treeWidget">
              <property name="columnCount">
               <number>3</number>
              </property>
              <column>
               <property name="text">
                <string notr="true">email</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">login</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">token</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="apps">
         <attribute name="title">
          <string>Apps</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="nodes">
         <attribute name="title">
          <string>Nodes</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTreeWidget" name="treeWidget_nodes">
              <property name="columnCount">
               <number>5</number>
              </property>
              <column>
               <property name="text">
                <string notr="true">nodename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">email</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">uuid</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>password</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>user</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="workflows">
         <attribute name="title">
          <string>Workflows</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="workers">
         <attribute name="title">
          <string>Workers</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="sessions">
         <attribute name="title">
          <string>Sessions</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="payloads">
         <attribute name="title">
          <string>Payloads</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="lost_pushpull_payloads">
         <attribute name="title">
          <string>lost pushpull payloads</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
